runtime/dart_isolate_unittests.cc

```diff
@@ -613,5 +613,39 @@ TEST_F(DartIsolateTest, DISABLED_ValidLoadingUnitSucceeds) {
   Wait();
 }
 
+TEST_F(DartIsolateTest, DartPluginRegistrantIsCalled) {
```

Comment:
Would it make sense to add an explicit test that there is no error when the class is absent?

Reply:
This would require a different location for the test. Since such tests require a new Dart main file, the Dart tooling complains about duplicated output file:

```txt
ERROR at //third_party/dart/build/dart/dart_action.gni:45:3: Duplicate output file.
  action(target_name) {
  ^--------------------
Two or more targets generate the same output:
  gen/flutter/runtime/assets/kernel_blob.bin

This is can often be fixed by changing one of the target names, or by
setting an output_name on one of them.

Collisions:
  //flutter/runtime:_dsk__ds_no_plugin_registrant_fixtures
  //flutter/runtime:_dsk__ds_runtime_fixtures

See //third_party/dart/build/dart/dart_action.gni:45:3: Collision.
  action(target_name) {
  ^--------------------
```

Any suggestion @chinmaygarde ?

Reply:
You could try to update `//testing/testing.gni` to support multiple Dart snapshot targets.  Or just create another testing target if it's easier - e.g. `executable("runtime_plugin_missing_unittests") { .... }` that depends on its own `test_fixtures("runtime_plugin_missing_fixtures")` which specifies a different Dart file.

Reply:
Actually it might be enough to just add another `test_fixtures` target and have the existing runtime executable depend on it.

Reply:
Ahh I see, that is probably what you did already. You'll want to update testing.gni so that you can tell it to use a different file name at line 159.

Reply:
Got it. Done